### DISCUSSION TITLE
Xion Class_Capstone Telebot

### DISCUSSION THREAD

**Discussion by awqigibran** [teamalgoritma/community#829] [Q&A]
Saya sedang mengerjakan capstone project Telebot dan sedang berada di Check Point 3 (Chatbot Functionality - Visualization report: /plot). Tidak ada issue untuk menyelesaikan panduan yang ada di Notebook Guide. Namun saya mengalami masalah ketika menjalankan bot_local.py, seharusnya ketika kita berikan prompt /plot maka chatbot akan merespon dengan memberikan pilihan unique_campaign_id, tapi fungsi tersebut tidak berjalan seperti ditunjukkan di snapshoot berikut

![Checkpoint 3 Problem](https://github.com/teamalgoritma/community/assets/150925883/573de634-bd2e-4fe1-83b3-723ffd5cf024)

Kode terkait saya lampirkan di snapshoot berikut. Sejauh saya coba trace, sepertinya tidak ada isu tetapi mungkin ada yang terlewat

![Checkpoint 3 Script Problem](https://github.com/teamalgoritma/community/assets/150925883/0916c38b-b7ff-4214-a626-b3cfc88bca68)

Apakah bisa dibantu kira-kira mana yang perlu saya cek dan perbaiki

Terima kasih banyak untuk bantuannya

**Comment by divakartika** [comment]
Dear @awqigibran 
Dari screenshot yang diberikan tidak terlihat ada kesalahan, kemungkinan kesalahan terletak pada bagian-bagian rumpang yang perlu diisi pada Checkpoint 3. Apakah setelah command `/plot` dikirimkan, tidak terdapat balasan "Choose campaign to be visualized"?

**Reply by awqigibran** [reply]
Mohon maaf baru membalas. Kode berjalan lancar setelah VS Code saya re-start. Kurang tahu penyebab pastinya kenapa.

Terima kasih
